proto/servicemanager/v3/servicemanager.proto

```diff
@@ -73,12 +73,18 @@ message VersionInfo {
     string description    = 3;
 }
 
+message NetworkParameters {
+    string subnet    = 1;
+    string ip        = 2;
+}
+
 message InstanceInfo {
-    InstanceIdent instance = 1;
-    uint32 uid             = 2;
-    uint64 priority        = 3;
-    string storage_path    = 4;
-    string state_path      = 5;
+    InstanceIdent instance               = 1;
+    NetworkParameters network_parameters = 2;
```

Comment:
move NetworkParameters to the bottom to support backward compatibility

Reply:
Done